docs/source/install.rst

```diff
@@ -1,6 +1,59 @@
 .. install
 
 Install MILES GUESS
--------------------
+===================
+First, set up a base Python environment on
```

Comment:
are any comments needed here about supported environments/versions of Python?

Reply:
I can add that.